documentation/site/pages/developers/smart-contracts/deployments.mdx

```diff
@@ -1,13 +1,31 @@
 # Deployments
```

Comment:
We should also capture the stake token address, which will be usdc, and USDC has different addresses on each chain